lib/travis/build/appliances/fix_hhvm_source.rb

```diff
@@ -6,18 +6,18 @@ module Build
     module Appliances
       class FixHhvmSource < Base
         def apply
-          sh.if "$(command -v lsb_release)" do
+          sh.if "$(command -v lsb_release) && $(lsb_release -cs) != precise" do
             command = <<-EOF
             if [[ -d /var/lib/apt/lists && -n $(command -v apt-get) ]]; then
               grep -l -i -r hhvm /etc/apt/sources.list.d | xargs sudo rm -vf
             fi
             EOF
             sh.cmd command, echo: false
             sh.cmd "apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0xB4112585D386EB94", echo: false, assert: false, sudo: true
-            sh.cmd 'add-apt-repository "deb [ arch=amd64 ] http://dl.hhvm.com/ubuntu $(lsb_release -sc) main"', echo: false, assert: false, sudo: true
+            sh.cmd 'add-apt-repository "deb [ arch=amd64 ] https://dl.hhvm.com/ubuntu $(lsb_release -sc) main"', echo: false, assert: false, sudo: true
```

Comment:
did you test on trusty as well again for this change to https?

Reply:
yeah, a trusty sudo run was included in the build referenced above

Reply:
Dope! I just realized I could have looked at the builds. 🤦‍♂️